GitUI/UserControls/RevisionGrid/Columns/RevisionGraphColumnProvider.cs

```diff
@@ -38,24 +35,7 @@ public RevisionGraphColumnProvider(RevisionGraph revisionGraph, IGitRevisionSumm
             };
         }
 
-        public RevisionGraphDrawStyleEnum RevisionGraphDrawStyle
-        {
-            get
-            {
-                if (_revisionGraphDrawStyle == RevisionGraphDrawStyleEnum.HighlightSelected)
-                {
-                    return RevisionGraphDrawStyleEnum.HighlightSelected;
-                }
-
-                if (AppSettings.RevisionGraphDrawNonRelativesGray)
-                {
-                    return RevisionGraphDrawStyleEnum.DrawNonRelativesGray;
-                }
-
-                return RevisionGraphDrawStyleEnum.Normal;
-            }
-            set { _revisionGraphDrawStyle = value; }
-        }
+        public RevisionGraphDrawStyle RevisionGraphDrawStyle;
```

Comment:
This makes it a public field, which is discouraged.
```suggestion
        public RevisionGraphDrawStyle RevisionGraphDrawStyle { get; set; }
```

Reply:
done